ferrocene/doc/internal-procedures/src/backports.rst

```diff
@@ -4,7 +4,7 @@
 Handling Backports
 ==================
 
-All development of Ferrocene happens in the ``main`` branch, but occasionally
+Most development of Ferrocene happens in the ``main`` branch, but occasionally
```

Comment:
```suggestion
All development of Ferrocene happens in the ``main`` branch, but occasionally
```

Reply:
hm, a revert of what I changed... commit message is "[we at times do dev on release branches when time is short](https://github.com/ferrocene/ferrocene/pull/1169/commits/50c80e03702c7424f5e0d75c6c1f122bd1be24ec)"

Reply:
We do not do dev on release branches.

Reply:
https://github.com/ferrocene/ferrocene/pull/988 is an example

it is rare, so maybe *most* is not strong enough

what about
> Ferrocene development normally happens in the main branch...

Reply:
Development does not mean making *any* change not strictly related to the development process. What this sentence mean is that all features or ticket work (development, not maintenance) happens on the main branch.

This commit should be reverted.

Reply:
Would talking about feature development help here? If we're saying "most" it is useful to talk about when not. 

It's also useful (especially in internal or internal ish dogs) to aim for clarity as well as correctness. If something is strictly true, but can be misunderstood, let's make it easier for the next person to understand.

Reply:
opened https://github.com/ferrocene/ferrocene/pull/1324